java/src/jmri/implementation/configurexml/MatrixSignalMastXml.java

```diff
@@ -30,7 +30,7 @@ public MatrixSignalMastXml() {
     public Element store(Object o) { // from mast p to XML
         MatrixSignalMast p = (MatrixSignalMast) o;
 
-        Element e = new Element("signalmast");
+        Element e = new Element("matrixsignalmast");
```

Comment:
No problem, see below. Could you perhaps restart the CI server to see if a 2nd run will pass?